.eslintrc.json

```
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "node": true,
+    "mocha": true,
+    "es6": false
+  },
+  "root": true,
+  "plugins": ["prettier"],
+  "parserOptions": {
+    "ecmaVersion": 6
+  },
+  "extends": ["eslint:recommended", "prettier"],
```

If you were to use say..
```suggestion
  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
```

You might be able to do away with the prettier invocation after eslint within `npm run lint`. This would also help avoid any conflict between prettier and eslint by embedding prettier's rules/logic within eslint itself.

Not sure how this interacts with the prettier-plugin-packagejson though.